README.Rmd

```diff
@@ -32,6 +38,91 @@ You can install the development version of ghqc from [GitHub](https://github.com
 pak::pak("a2-ai/ghqc")
 ```
 
-## Short Tour
+## Interacting with ghqc
+
+``` r
+library(ghqc)
+```
+
+## Configuration Information Repository
+
+The ghqc ecosystem has been developed with the intent organizations can customize elements of the QC process. Currently, the customizable elements are:
+
+- **Checklists**: yaml files contained within a directory named "checklist" *(required)*
+
+- **Prepended Note**: A note to prepend to all checklists named "note" *(optional)*
+
+- **Logo**: A logo to be included in the header of the resultant record's header named as "logo.png"
```

Comment:
the logo is required, right? add *(required)*

Reply:
https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L55

Done!